pkg/utils/report/labels.go

```diff
@@ -19,16 +19,17 @@ import (
 const (
 	LabelDomain = "kyverno.io"
 	//	resource labels
-	LabelResourceHash      = "audit.kyverno.io/resource.hash"
-	LabelResourceUid       = "audit.kyverno.io/resource.uid"
-	LabelResourceGVR       = "audit.kyverno.io/resource.gvr"
-	LabelResourceNamespace = "audit.kyverno.io/resource.namespace"
-	LabelResourceName      = "audit.kyverno.io/resource.name"
+	LabelResourceHash           = "audit.kyverno.io/resource.hash"
+	LabelResourceUid            = "audit.kyverno.io/resource.uid"
+	LabelResourceGVR            = "audit.kyverno.io/resource.gvr"
+	AnnotationResourceNamespace = "audit.kyverno.io/resource.namespace"
```

Comment:
Was there an upstream Kyverno change like this to this?

Reply:
The change wasnt done in a single PR, the issue no longer exists in 1.11.0 as we use audit.kyverno.io/resource.uid to label the admissionreports. I implemented and tested the same in n4k.